# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-pcmcia-ppc.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.8/drivers/pcmcia/Kconfig.vanilla	2024-05-07 00:02:12.985063114 +0200
+++ linux-6.8/drivers/pcmcia/Kconfig	2024-05-07 00:02:29.760064454 +0200
@@ -230,7 +230,7 @@
 
 config PCMCIA_PROBE
 	bool
-	default y if ISA && !ARCH_SA1100 && !PARISC
+	default y if ISA && !ARCH_SA1100 && !PARISC && !PPC
 
 config OMAP_CF
 	tristate "OMAP CompactFlash Controller"
